Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM fuzzers/afl:2.52
```

Comment:
Same here. Avoid git clone, and perhaps commit the corpus directly rather than calling wget every time.